lesson_7/alexandra_siriciuc/homework decorators.py

```diff
@@ -0,0 +1,60 @@
+def uppercase_decorator(function):
+        def inner_function(arg):
+            func = function(arg)
```

Comment:
This works as expected, except that the variable names are a bit deceiving.
Here you have a variable called func, when this is actually the string returned by the call of the original function. Please rename to something that better reflects the data it holds.

Reply:
@AlexandraSRC am marcat comentariul lui Marius de mai sus cu fisierul de assignment resolved. Dupa ce modifici numele variabilei putem sa facem merge la tema.